cmd/gadgetctl/main.go

```diff
@@ -27,6 +27,7 @@ import (
 	commonutils "github.com/inspektor-gadget/inspektor-gadget/cmd/common/utils"
 	_ "github.com/inspektor-gadget/inspektor-gadget/pkg/all-gadgets"
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/environment"
+	_ "github.com/inspektor-gadget/inspektor-gadget/pkg/operators/cli"
```

Comment:
I didn't understand what is the operators/cli. But could we have an explicit Init function instead of relying on init()?

Related to https://github.com/inspektor-gadget/inspektor-gadget/issues/1992

Reply:
CLI is the operator that prints out the results. When we add support for multiple targets (external logging servers, etc.) all of these operators become effectively data sinks that data can be routed to - so the user could chose dynamically which operator should work on which datasource.